terratest/src/test/pega/pega-tier-deployment_test.go

```diff
@@ -187,7 +187,7 @@ func VerifyDeployment(t *testing.T, pod *k8score.PodSpec, expectedSpec pegaDeplo
 	//require.Equal(t, int32(10), pod.Containers[0].StartupProbe.TimeoutSeconds)
```

Comment:
Should these still be commented out?

Reply:
The test code is commented out due to an issue of our test infra that doesn't really allow us to run tests against different k8s server versions.